gensim/test/test_fasttext.py

```diff
@@ -206,16 +205,16 @@ def test_load_fasttext_new_format(self):
         # vector for oov words are slightly different from original FastText due to discarding unused ngrams
         # obtained using a modified version of ./fasttext print-word-vectors lee_fasttext_new.bin
         expected_vec_oov = [
-            -0.53378,
-            -0.19,
-            0.013482,
-            -0.86767,
-            -0.21684,
-            -0.89928,
-            0.45124,
-            0.18025,
-            -0.14128,
-            0.22508
+            -0.49111,
```

Comment:
Ditto:
What is the reason for this change? Could it because of the `len(ngrams)` issue mentioned in a comment above?